src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -821,11 +821,11 @@ internal ResultVersionOne CreateResultVersionOne(Result v2Result)
             return result;
         }
 
-        internal static RuleVersionOne CreateRuleVersionOne(IRule v2IRule)
+        internal static RuleVersionOne CreateRuleVersionOne(MessageDescriptor v2IRule)
```

Comment:
>v2IRule [](start = 78, length = 7)

`v2MessageDescriptor`

Reply:
Done and searched for other linger IRule references in names, none found.

---
In reply to: [255310270](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255310270) [](ancestors = 255310270)